src/handlers/notify_zulip.rs

```diff
@@ -12,6 +12,8 @@ pub(super) struct NotifyZulipInput {
     /// For example, if an `I-prioritize` issue is closed,
     /// this field will be `I-prioritize`.
     label: Label,
+    is_default_valid: bool,
+    names: Vec<String>,
```

Comment:
maybe `include_config_names`?
```suggestion
    include_config_names: Vec<String>,
```

Reply:
Done.